jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java

```diff
@@ -42,17 +42,36 @@ public A notEmpty() {
     }
 
     /**
-     * Match passed value with table size
+     * Match passed value with table rows count
+     * @param condition to compare
+     */
+    @JDIAction("Assert that '{name}' count {0}")
+    public A count(Matcher<Integer> condition) {
+        jdiAssert(table().count(), condition);
+        return (A) this;
+    }
+
+    /**
+     * Match passed value with table rows count
+     * @param count to compare
+     */
+    @JDIAction("Assert that '{name}' count {0}")
+    public A count(int count) {
+        return count(Matchers.is(count));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818435)